samples/zigbee/light_switch/include/nus_cmd.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file nus_cmd.h
+ *
+ * @brief APIs for NUS Command Service.
+ */
+
+#ifndef __NUS_CMD_H__
+#define __NUS_CMD_H__
+
+/** @brief Type indicates function called when Bluetooth LE connection
```

Comment:
I personally prefer a `function prototype` instead of `type indicates function`. Feel free to keep it as is :slightly_smiling_face: